ui/app/styles/core/table.scss

```diff
@@ -86,6 +86,30 @@
       overflow: hidden;
       text-overflow: ellipsis;
       white-space: nowrap;
+
+      &.is-hover-overflowable {
+        position: relative;
+
+        .hover-overflow {
+          display: none;
+          position: absolute;
+          top: 0;
+          left: 0;
+          padding: calc(1.25em - 1px) 1.5em;
+        }
```

Comment:
This is less than delightful but apart from stating the padding in pixels (17) I couldn’t figure out another way.

`.is-hover-overflowable` is another questionable name, easily changed!

Reply:
I agree the math is a little awkward, but it's a true expression of what's happening and will be resilient in the event that font-size changes, so I like it.

`.is-hover-overflowable` is a mouthful, but honestly fine. Maybe `.is-revealable`? It's little more succinct and doesn't conflate the behavior with the interaction. Idk.